src/mesh.rs

```diff
@@ -371,6 +381,25 @@ impl<'a> Primitive<'a> {
         self.json
     }
 
+    /// Returns the bounds (min/max) of the POSITION attribute if there is one, otherwise `None`.
+    /// May panic for invalid glTF files. Use json::validation::Validate::validate_minimally
+    /// to handle this gracefully.
+    pub fn bounds(&self) -> Option<Bounds> {
```

Comment:
[It's no less convenient.](https://play.rust-lang.org/?gist=7a7f0acc9d2b06dd3c221f736899a3a4&version=stable)